changelogs/current.yaml

```diff
@@ -408,6 +408,11 @@ new_features:
   change: |
     Add the option to reduce the rate limit budget based on request/response contexts on stream done.
     See :ref:`apply_on_stream_done <envoy_v3_api_field_config.route.v3.RateLimit.apply_on_stream_done>` for more details.
+- area: router
```

Comment:
since histograms are expensive to compute, and this one will be computed per request per cluster, have you considered having a config flag to enable this histogram when needed? I now don't remember if we have any existing mechanism that tells Envoy which histograms to initialize at startup, maybe @jmarantz knows